libs/wallet/src/vega-transaction-dialog/vega-transaction-dialog.tsx

```diff
@@ -1,155 +1,94 @@
-  useEffect(() => {
-    if (transaction.status !== VegaTxStatus.Default || finalizedOrder) {
-      setOrderDialogOpen(true);
-    } else {
-      setOrderDialogOpen(false);
-    }
-  }, [finalizedOrder, setOrderDialogOpen, transaction.status]);
-
+  const computedIntent = intent ? intent : getIntent(transaction);
+  const computedTitle = title ? title : getTitle(transaction);
+  const computedIcon = icon ? icon : getIcon(transaction);
+  // Each dialog can specify custom dialog content using data returned via
+  // the subscription that confirms the transaction. So if we get a success state
+  // and this custom content is provided, render it
+  const content =
+    transaction.status === VegaTxStatus.Complete && children ? (
+      children
+    ) : (
+      <VegaDialog transaction={transaction} />
+    );
   return (
     <Dialog
-      open={orderDialogOpen}
-      onChange={(isOpen) => {
-        setOrderDialogOpen(isOpen);
-
-        // If closing reset
-        if (!isOpen) {
-          reset();
-        }
-      }}
-      intent={getDialogIntent(finalizedOrder, transaction)}
+      open={isOpen}
+      onChange={onChange}
+      intent={computedIntent}
+      title={computedTitle}
+      icon={computedIcon}
     >
-      <VegaDialog
-        key={`${title.toLowerCase().split(' ').join('-')}-tx-${
-          transaction.txHash
-        }`}
-        transaction={transaction}
-        finalizedOrder={finalizedOrder}
-        title={title}
-        children={children}
-      />
+      {content}
     </Dialog>
   );
 };
 
 interface VegaDialogProps {
   transaction: VegaTxState;
-  finalizedOrder: Order | null;
-  title: string;
-  children?: ReactNode;
 }
 
-export const VegaDialog = ({
-  transaction,
-  finalizedOrder,
-  title,
-  children,
-}: VegaDialogProps) => {
+/**
+ * Default dialog content
+ */
+export const VegaDialog = ({ transaction }: VegaDialogProps) => {
   const { VEGA_EXPLORER_URL } = useEnvironment();
-  const headerClassName = 'text-h5 font-bold text-black dark:text-white';
-
-  if (children && transaction.status === VegaTxStatus.Default) {
-    return <div>{children}</div>;
-  }
 
-  // Rejected by wallet
   if (transaction.status === VegaTxStatus.Requested) {
     return (
-      <OrderDialogWrapper
-        title="Confirm transaction in wallet"
-        icon={<Icon name="hand-up" size={20} />}
-      >
-        <p>
-          {t(
-            'Please open your wallet application and confirm or reject the transaction'
-          )}
-        </p>
-      </OrderDialogWrapper>
+      <p data-testid={transaction.status}>
+        {t(
+          'Please open your wallet application and confirm or reject the transaction'
+        )}
+      </p>
     );
   }
 
-  // Transaction error
   if (transaction.status === VegaTxStatus.Error) {
     return (
-      <OrderDialogWrapper
-        title="Order rejected by wallet"
-        icon={<Icon name="warning-sign" size={20} />}
-      >
+      <div data-testid={transaction.status}>
```

Comment:
ok, so I have tried to follow the [designs]( https://www.figma.com/file/K49Nx7XfocShhHcwwQzQkm/Console-V2?node-id=1652%3A26053) in Figma. we need to be aware we are changing these as we can see. 



![image](https://user-images.githubusercontent.com/16125548/180597243-5ac618ba-4cc0-4917-9c91-7b2ac79399e8.png)


<img width="1225" alt="Transaction Complete" src="https://user-images.githubusercontent.com/16125548/180597226-1dd34327-bd2b-449a-963b-8b63636e74bb.png">


<img width="1211" alt="VEGA TRADING PORTFOLIO" src="https://user-images.githubusercontent.com/16125548/180597222-eeacf997-113b-445b-a2db-cf83c3babe04.png">